### ISSUE TITLE
[BUG] Java 21 Bug In 32 Bit

### ISSUE THREAD

**Issue by SyaripMC** [Vera-Firefly/PojavLauncher-Beta-Zink-CN#330]
### Describe the bug

I Want To Try Using Java 21 On My 32 Bit Device Using Minecraft Java 1.21. When I tried there was a bug from the panorama / background and Cannot enter server/world because it is stuck loading . Please fix the bug as soon as possible, thank you.

### The log file and images/videos

![Screenshot_2024-11-29-20-20-11-537_net kdt pojavlaunch firefly](https://github.com/user-attachments/assets/1f8386bf-965f-4a7e-8307-164ace0dcc4f)

### Steps To Reproduce

```markdown
1. Start PojavLauncher
2. There is a bug in the panorama / background
3. Trying to Play Server
4. Turns out it's stuck loading
```

### Expected Behavior

Pls fix it so i can play minecraft java 1.21

### Platform

```markdown
- Device model: Redmi 10A
- CPU architecture: arm32 (32 bit)
- Android version: 11
- PojavLauncher version: fuchsia-20241117-a833e59-main_v3
```

**Comment by aaaapai** [comment]
哦，这是一个很早就有的问题，它不单单体现在PGW上

**Comment by FathirY** [comment]
Up, please fix this, i only have 32bit devices

**Comment by Vera-Firefly** [comment]
> ### Describe the bug
> I Want To Try Using Java 21 On My 32 Bit Device Using Minecraft Java 1.21. When I tried there was a bug from the panorama / background and Cannot enter server/world because it is stuck loading . Please fix the bug as soon as possible, thank you.
> 
> ### The log file and images/videos
> ![Screenshot_2024-11-29-20-20-11-537_net kdt pojavlaunch firefly](https://private-user-images.githubusercontent.com/173051439/391210145-1f8386bf-965f-4a7e-8307-164ace0dcc4f.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwNjM5ODgsIm5iZiI6MTczMzA2MzY4OCwicGF0aCI6Ii8xNzMwNTE0MzkvMzkxMjEwMTQ1LTFmODM4NmJmLTk2NWYtNGE3ZS04MzA3LTE2NGFjZTBkY2M0Zi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwMVQxNDM0NDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZjg1Njk0Y2JlN2NiZDlhZDdmOGQzZWI5N2IzOGM2NzA1NzQ2Y2UxOGQyNDM2ZWQzNDFjMzE2ZDkzMjI1Njk5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.QaNtzfpR9HOY1PDPY_CFhMFjF8uuiZIYJIULjtvj7XU)
> 
> ### Steps To Reproduce
> ```
> 1. Start PojavLauncher
> 2. There is a bug in the panorama / background
> 3. Trying to Play Server
> 4. Turns out it's stuck loading
> ```
> 
> ### Expected Behavior
> Pls fix it so i can play minecraft java 1.21
> 
> ### Platform
> ```
> - Device model: Redmi 10A
> - CPU architecture: arm32 (32 bit)
> - Android version: 11
> - PojavLauncher version: fuchsia-20241117-a833e59-main_v3
> ```
> 
> ### Anything else?
> _No response_

Duplicate Submission(https://github.com/Vera-Firefly/Pojav-Glow-Worm/issues/281)
